src/analyzers/cha-analyzer.ts

```
@@ -0,0 +1,600 @@
+    
+    const setAccessors = classDecl.getSetAccessors();
+    for (const setter of setAccessors) {
+      const methodInfo = this.extractMethodInfo(setter, className);
+      if (methodInfo) {
+        node.methods.push(methodInfo);
+      }
+    }
+    
+    // Extract constructor
+    const constructors = classDecl.getConstructors();
+    for (const constructor of constructors) {
```

Biome (1.9.4)

[error] Do not shadow the global "constructor" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)